src/Products/Formulator/zeamsupport.py

```diff
@@ -23,10 +25,10 @@ def decode(value):
     """Helper to ensure we have unicode everywhere, as Formulator use
     it in an optional fashion.
     """
-    if not isinstance(value, unicode):
+    if not isinstance(value, six.text_type):
         if isinstance(value, str):
             return value.decode('utf-8')
```

Comment:
Should we be more clearly using `bytes` here?
```suggestion
    if not isinstance(value, six.text_type):
        if isinstance(value, bytes):
            return value.decode('utf-8')
```